aws-terraform/README.md

```diff
@@ -23,16 +23,25 @@ Init mode will guide the administrator through an interactive process which gene
 
 ## OP-SCIM Bridge 
 
-The 1Password SCIM Bridge is distributed as the Debian package and can be installed during the deployment process using the following repo configuration: `deb https://apt.agilebits.com/op-scim/ stable op-scim`. [Repo GPG key](apt.agilebits.com_repo_key.asc).  
+The 1Password SCIM Bridge is distributed as the Debian package and can be installed during the deployment process using the following repo configuration: `deb https://apt.agilebits.com/op-scim/ stable op-scim`. [Repo GPG key](https://apt.agilebits.com/gpg.key) can be downloaded from the same repo.  
+Package upgrade script is included in the instance [User Data](https://github.com/1Password/scim-examples/blob/81f66e808941a9215af3fb27b3f4351c9c8b17ff/aws-terraform/terraform/module_scim_app/data/user_data/02-environment.yml#L13) and instance cron job runs that script hourly to automatically install newer version when it is available. Automatic updates can be disabled by removing the cron job:  
+```
+sudo rm /etc/cron.d/50_op-scim
+```  
+
```

Comment:
I think we should check for updates every day or at most every hour, otherwise frequent updates will cost us a lot of money.
Or have it configured in `/etc/defaults/op-scim` file

Reply:
current configuration checks for updates hourly:
```
# op-scim package upgrade cron job, runs hourly
    - content: '0 * * * * root /usr/local/bin/op-scim-upgrade.sh 2>&1 | logger -t op-scim-upgrade
```

Reply:
current configuration checks for updates hourly:
```
# op-scim package upgrade cron job, runs hourly
    - content: '0 * * * * root /usr/local/bin/op-scim-upgrade.sh 2>&1 | logger -t op-scim-upgrade
```

I would keep 1 hour interval here.